src/librustc_typeck/diagnostics.rs

```diff
@@ -1535,7 +1556,6 @@ register_diagnostics! {
     E0219, // associated type defined in higher-ranked supertrait
     E0220, // associated type not found for type parameter
     E0221, // ambiguous associated type in bounds
-    E0222, // variadic function must have C calling convention
```

Comment:
Leave this line in, commented out with an indicator that it's been merged with E0045